geopy/geocoders/osm.py

```diff
@@ -114,6 +114,7 @@ def geocode(
             self,
             query,
             exactly_one=True,
+            limit=None,
```

Comment:
Could you please make this arg the latest one? [The base method](https://github.com/geopy/geopy/blob/d3e076a8ce0cf47667af36d28c40d24039ac516f/geopy/geocoders/base.py#L205) has `timeout` as its last arg and this order should be preserved to not cause a strange behaviour when `geocode` is called using positional args.

Also it would be nice to add a docstring here (with `versionadded:: 1.13.0`). You can borrow one [from Photon, for example](https://github.com/geopy/geopy/blob/d3e076a8ce0cf47667af36d28c40d24039ac516f/geopy/geocoders/photon.py#L101).

Reply:
You mean limit should be placed after timeout, is that right ?

Reply:
After `timeout` or as the last one (after `geometry`) – both options are fine.

Reply:
Done in cd9eea9